unit-tests/utilities/time/test-work_week-class.cpp

```diff
@@ -0,0 +1,24 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+// Unit Test Goals:
+// Test the work_week class
+
+//#cmake:add-file ../../../common/utilities/time/work_week.h
+//#cmake:add-file ../../../common/utilities/time/work_week.cpp
+
+
+#include "common.h"
+#include "../common/utilities/time/work_week.h"
+
+using namespace utilities::time;
+
+// Test description:
+// > Test the work_week main functions
```

Comment:
Can test expected behavior:
* Day 1 is ww1, day 7 is ww1, day 8 is ww2
* Year with 53 weeks, and subtraction with ww53
...